.github/workflows/cd-prestabulle-clean.yml

```diff
@@ -0,0 +1,37 @@
+name: APIQANIGHTLY PRESTABULLE CLEANING
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - develop
+
+env:
+  TF_VERSION: "1.3.7"
+  LABELS: ${{toJSON(github.event.pull_request.labels)}}
+
+jobs:
+  apiqanightly_prestabulle:
+    name: apiqanightly Cleaning Prestabulle
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+    if: contains(github.event.pull_request.labels.*.name, 'prestabulle1') ||
+      contains(github.event.pull_request.labels.*.name, 'prestabulle2') ||
+      contains(github.event.pull_request.labels.*.name, 'prestabulle3')
+
+    steps:
+      - name: Checkout the repository 🎁
+        uses: actions/checkout@v3
+
+      - name: Setting up the Bulle variable for later 💬
+        run: echo "BULLE=$BULLE$(echo $LABELS > labels.json && cat labels.json | jq -r '.[]|.name'|grep prestabulle)" >> $GITHUB_OUTPUT
+        id: get_bulle
+
+      - name: Trigger clean prestabulle
+        uses: aurelien-baudet/workflow-dispatch@v2
+        with:
+          workflow: cd-prestabulle-clean.yml
+          repo: PrestaShopCorp/qanightlyresults-cd
```

Comment:
:warning: Please don't link to external/private repositories.

Could you move and open `PrestaShopCorp/qanightlyresults-cd` to the `PrestaShop` org ?